registration/api/v2/views.py

```diff
@@ -292,6 +299,10 @@ def image(self, request, pk: str):
                 application.logo = request.data['logo']
                 application.save()
 
+            if 'hero_cover_image' in request.data:
```

Comment:
Does the current implementation allows erasing the image? Can we do a similar thing to the other endpoints, where sending an empty message erases the stored value?

Reply:
@giovannicimolin, I have implemented the changes to allow deleting the image from the field. However, Django doesn't delete the image from the storage backend and recommends creating a cron job to delete unused images.

Please check the testing instructions and the code changes for the details.

Reply:
@lgp171188 Thanks for implementing this! Looks good.